docs/class4/module1/1.0-explore.rst

```diff
@@ -22,6 +26,22 @@ If prompted, select yes for trusting the ssh connection in your terminal.
    [centos@nginx-manager ~]$ pwd
    /home/centos
 
+Option 2 - Web Shell
+====================
+
+.. image:: ./UDF-select-webshell.png
+
+Once the web shell opens up enter the following commands to switch to the 
+``centos`` user and also to set the Terminal for editing.
+
+.. code-block:: shell-session
+
+   [root@nginx-manager /]# su centos                                                                                      
```

Comment:
this should be su - centos, otherwise it does not load the user env variables, it keeps root.